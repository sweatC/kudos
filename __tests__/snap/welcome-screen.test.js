import React from 'react';
import WelcomeScreen from '../../screens/welcome-screen';
import renderer from 'react-test-renderer';


it('Welcome screen renders without crashing', () => {
    const rendered = renderer.create(<WelcomeScreen />).toJSON();
    expect(rendered).toMatchSnapshot();
});