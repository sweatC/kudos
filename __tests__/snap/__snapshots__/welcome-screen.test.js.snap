// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome screen renders without crashing 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "flex": 4,
        "resizeMode": "center",
      }
    }
  />
  <View
    style={
      Object {
        "flex": 2,
        "marginTop": 40,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "hsla(52, 75%, 6%, 0.91)",
          "fontFamily": "Roboto",
          "fontSize": 28,
          "fontWeight": "400",
          "textAlign": "center",
        }
      }
    >
      Kudos
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "hsla(52, 75%, 6%, 0.91)",
          "fontFamily": "Roboto",
          "fontSize": 14,
          "fontWeight": "100",
          "marginTop": 2,
          "textAlign": "center",
        }
      }
    >
      Your small gestures grtitude
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 3,
        "marginTop": 15,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "hsla(52, 75%, 6%, 0.91)",
          "borderRadius": 5,
          "height": 45,
          "width": 250,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "#fff",
            "marginTop": 12,
            "textAlign": "center",
          }
        }
      >
        Sign in
      </Text>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      onPress={[Function]}
      style={
        Object {
          "color": "hsla(52, 75%, 6%, 0.91)",
          "fontFamily": "Roboto",
          "fontSize": 10,
          "marginTop": 7,
          "textAlign": "center",
        }
      }
    >
      Don't have an account? Sign up
    </Text>
  </View>
</View>
`;
